### DISCUSSION TITLE
Towncrier release

### DISCUSSION THREAD

**Discussion by asvetlov** [twisted/towncrier#364] [Q&A] [answered]
Is there an estimation for the new final release?

21.3.0 is broken if `title_format` is customized in config file.
21.9.0rc1 has a fix for it, it would be nice to have this fix finally released!

**Comment by adiroiban** [comment, marked as answer]
there is a PR for a release  at 
https://github.com/twisted/towncrier/pull/360

Give it a try and if all is ok we can do the final release

Thanks!
